specification/app/resource-manager/Microsoft.App/preview/2025-10-02-preview/ContainerAppFunctions.json

```diff
@@ -0,0 +1,327 @@
+            "schema": {
+              "$ref": "#/definitions/ContainerAppFunctionCollection"
+            }
+          },
+          "default": {
+            "description": "Common error response.",
+            "schema": {
+              "$ref": "./CommonDefinitions.json#/definitions/DefaultErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "List Container App's functions": {
+            "$ref": "./examples/ContainerAppFunctions_List.json"
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.App/containerApps/{containerAppName}/functions/{functionName}": {
+      "get": {
+        "tags": [
+          "AzureFunctionsOnContainerApps"
+        ],
+        "summary": "Get a specific function of a Container App from the latest Revision.",
+        "operationId": "ContainerAppFunctions_GetFunction",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "name": "containerAppName",
+            "in": "path",
+            "description": "Name of the Container App.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "functionName",
+            "in": "path",
+            "description": "Name of the Function.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/ContainerAppFunction"
+            }
+          },
+          "default": {
+            "description": "Common error response.",
+            "schema": {
+              "$ref": "./CommonDefinitions.json#/definitions/DefaultErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Get Container App's function": {
+            "$ref": "./examples/ContainerAppFunctions_Get.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "ContainerAppFunction": {
+      "description": "Container App Function.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v5/types.json#/definitions/ProxyResource"
+        }
+      ],
+      "properties": {
+        "location": {
+          "type": "string",
+          "description": "The geo-location where the resource lives",
+          "readOnly": true
+        },
```

Comment:
As you want to have location property, what about using `TrackedResource` https://github.com/Azure/azure-rest-api-specs/blob/e5e776854ab5d10da58d1d36fe8521e33d70e510/specification/common-types/resource-management/v5/types.json#L494C6-L494C21